tests/migrations/test_optimizer.py

```diff
@@ -631,3 +632,34 @@ def test_optimize_through_fields(self):
                 migrations.CreateModel("Bar", [("width", models.IntegerField())]),
             ],
         )
+
+    def test_optimize_elidable_operation(self):
+        self.assertOptimizesTo(
+            [
+                operations.special.ElidableOperation(elidable=True),
```

Comment:
I think the test only fails in this case with your suggested implementation of `elidable`. The only attribute check for `elidable` in this implementation is performed by `ElidableOperation` on its instances.